### ISSUE TITLE
Test Issue: How Many People Can Be Pinged in a Single Issue?

### ISSUE THREAD

**Issue by mchowning** [wordpress-mobile/gutenberg-mobile#3507]
## TLDR: You can safeiy ignore this issue/ping. Sorry for the noise. Go ahead and unsubscribe from this issue.

The only purpose of this issue is to test how many people I can ping in a single issue. There appears to be a limit where I can only mention and ping 50 people at a time, but I don't know if that limit is per issue or per comment. #doing-science-sort-of. I'm looking into this because I need to find a way to ping >800 people. Additional context if you're interested: p1620939359333200-slack-C0HTU7HV3.

1. @bjtitus
2. @frosty
3. @koke
4. @bummytime
5. @shiki
6. @AliSoftware
7. @astralbodies
8. @pmusolino
9. @zwarm
10. @Ecarrion
11. @oguzkocer
12. @khaykov
13. @emilylaguna
14. @renanferrari
15. @planarvoid
16. @designsimply
17. @jkmassel
18. @jleandroperez
19. @iamthomasbishop
20. @mokagio
21. @yaelirub 
22. @mattmiklic
23. @ashiagr
24. @aerych
25. @0nko
26. @allendav
27. @ScoutHarris
28. @diegoreymendez
29. @ceyhun
30. @guarani
31. @jaclync
32. @ctarda
33. @SiobhyB
34. @ealeksandrov
35. @nbradbury
36. @loremattei
37. @geriux
38. @elibud
39. @wzieba
40. @ThomazFB
41. @JavonDavis
42. @mzorz
43. @belcherj
44. @leandroalonso
45. @Stojdza
46. @mkevins
47. @rachelmcr
48. @aforcier
49. @ParaskP7
50. @thehenrybyrd 
51. @illusaen 
52. @AmandaRiu 
53. @fluiddot 
54. @mattc6246

**Comment by mchowning** [comment]
1. @enejb 
2. @mattc6246
3. @cameronvoell 
4. @chipsnyder

**Comment by shiki** [comment]
<img src="https://i.imgur.com/niHYD3m.jpg" width="400">

**Comment by emilylaguna** [comment]
<img src="https://media.giphy.com/media/CJ6namLBWqeAw/giphy.gif" />
